_events/2021/2021-07-dei-speaker-series-ncwit.md

```diff
@@ -1,34 +1,26 @@
 ---
-title: "POSTPONED: NCWIT Event"
-subtitle: The National Center for Women & Information Technology (NCWIT) - US-RSE DEI-WG Speaker Series
-expires: 2021-08-31
-event_date: "TBD, 2021"
+title: "The National Center for Women & Information Technology (NCWIT)"
```

Comment:
should DEI-WG speaker series be part of the title? or at least something like "talk"?

Reply:
now I see it's in the subtitle - maybe that's enough?

Reply:
I think the subtitle is sufficient for now. New website with slightly less jumbo font and more flexibility coming soon...